glossary.yml

```diff
@@ -5726,6 +5726,10 @@
     term: ግጥም ጥርብ ግራፍ
     def: >
       የቁጥር ውሂብ ስብስብ ስርጭት ግራፊክ ውክልና ፣ ብዙውን ጊዜ ቀጥ ያለ የአሞሌ ግራፍ
+ uk:
+   term: "гістограма"
+   def: >
+    Спосіб графічного представлення табличних даних; кількісні співвідношення деякого показника представлені у вигляді прямокутників, площі яких пропорційні.
```

Comment:
This is tolerable, but we recommend to not to have very long lines. About 70-80 characters is optimal.